core/api/api.hpp

```diff
@@ -309,13 +309,6 @@ namespace fc::api {
 
     API_METHOD(NetAddrsListen, PeerInfo)
 
-    API_METHOD(PaychVoucherAdd,
```

Comment:
It's required for miner, keep it.

Reply:
Can miner use PaychannelManager directly? There is a self dependency in case API uses PaychannelManager.

Reply:
done